content/about.md

```diff
@@ -3,58 +3,57 @@ id: about
 
-- Mentees receive expert advice on their projects, goals, and the community.
-- Shorter learning curve through the mentorship relationship.
-- Members are empowered to acquire important social, organizational, and technical skills.
-- Mentors grow their expertise by guiding members of the community.
-- An opportunity to share experience and knowledge.
-- Learning and leadership opportunity.
+Galaxy is a sprawling ecosystem and a dispersed community. It is easy for newcomers to get overwhelmed by the many Galaxy resources and community communication channels. The Galaxy Mentor Network is established to connect newcomers with experienced community members to help prevent newcomers from getting lost.
 
-## Application Guidelines
+<div class="img-fluid mx-auto trim-p">
 
-### Eligibility
+![The Galaxy Community at GCC2019](assets/images/gcc2019-grouppicture.jpg)
 
-The Galaxy Community Mentorship program is open to everyone who meets the following requirements:
+<small>GCC2019 in Freiburg, Germany, photo by Bérénice Batut</small>
+</div>
 
-### GMentee:
+## Galaxy Mentoring Network (GMN)
 
-- You are a member of the community i.e member of one of our channels.
-- You have an intended project or you are willing to take on one of the projects suggested to you by your mentor.
-- You are willing to take full responsibility for the completion of your project.
-- GMentees will be required to devote about 5 hours weekly to their projects.
-- Bi-weekly calls of about 30 minutes will be held between mentees and mentors.
+Galaxy Mentoring Network (GMN) program is an initiative by the Galaxy Community to build her members in open source projects.
 
-### GMentor:
+It is run on an open enrollment basis where mentees and mentors can apply at any time of need, this is so especially because we have a lot of new members who join regularly and may immediately need someone to guide them. This is a program by Galaxy members and for Galaxy members.
 
-- You are an active member of the community (for example, in the channels, contributing to the Galaxy GitHub repositories, events, e.t.c)
-- Willing to give detailed and timely reviews of the projects and contributions of your GMentees.
-- You are willing to meet with the GMentee on a biweekly 30 minutes call.
-- GMentors will be required to mentor for at least 8 weeks.
-- GMentors will be expected to recommend helpful projects if the GMentee has no intended projects during application.
+## Vision
+
+We hope to increase the proficiency of members in the use of Galaxy tools and servers, contributing to open source projects as well as becoming active members of any of our channels and working groups.
 
-### Matching Process:
```

Comment:
Where will be all the content you deleted there?

Reply:
Good morning! I moved the content to `application.md`.